libs/features/quest/src/quest-template-locale/quest-template-locale.component.html

```diff
@@ -0,0 +1,104 @@
+            <div class="form-group col-12 col-sm-12 col-md-8 col-lg-9 col-xl-12">
+              <label class="control-label" for="id">CompletedText</label>
+              <textarea [formControlName]="'CompletedText'" id="completed-text" class="form-control form-control-sm"></textarea>
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-6 col-lg-6 col-xl-6">
+              <label class="control-label" for="id">ObjectiveText1</label>
+              <input [formControlName]="'ObjectiveText1'" id="objective-text-1" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-6 col-lg-6 col-xl-6">
+              <label class="control-label" for="id">ObjectiveText2</label>
+              <input [formControlName]="'ObjectiveText2'" id="objective-text-2" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-6 col-lg-6 col-xl-6">
+              <label class="control-label" for="id">ObjectiveText3</label>
+              <input [formControlName]="'ObjectiveText3'" id="objective-text-3" class="form-control form-control-sm" />
+            </div>
+            <div class="form-group col-12 col-sm-6 col-md-6 col-lg-6 col-xl-6">
+              <label class="control-label" for="id">ObjectiveText4</label>
+              <input [formControlName]="'ObjectiveText4'" id="objective-text-4" class="form-control form-control-sm" />
+            </div>
+            <div class="col-12 form-error" [hidden]="editorService.isFormIdUnique()">
+              <i class="fas fa-exclamation-triangle"></i>
+              <span [innerHTML]="'UNIQUE' | translate: { ENTITY_SECOND_ID_FIELD: editorService.entitySecondIdField }"></span>
+            </div>
+          </div>
+        </form>
+        <div class="row">
+          <div class="col-12">
+            <keira-editor-buttons [editorService]="editorService" />
+            <ngx-datatable
+              id="editor-table"
+              class="bootstrap table table-striped text-center datatable-select"
+              [rows]="editorService.newRows"
+              [headerHeight]="DTCFG.headerHeight"
+              [footerHeight]="DTCFG.footerHeight"
+              [columnMode]="DTCFG.columnMode"
+              [rowHeight]="DTCFG.rowHeight"
+              [selectionType]="DTCFG.selectionType"
+              (select)="editorService.onRowSelection($event)"
+              (keydown.delete)="editorService.deleteSelectedRow()"
+            >
+              <ngx-datatable-column [minWidth]="30" [maxWidth]="30" [sortable]="false">
+                <ng-template let-row="row" ngx-datatable-cell-template>
+                  <i [hidden]="!editorService.isRowSelected(row)" class="fas fa-chevron-right"></i>
+                </ng-template>
+              </ngx-datatable-column>
+              <ngx-datatable-column name="Id" prop="ID" [minWidth]="80" [maxWidth]="150"></ngx-datatable-column>
```

Comment:
This can be shortened to:
<ngx-datatable-column name="Id" prop="ID" [minWidth]="80" [maxWidth]="150"/>